src/pages/PersDataProt.js

```diff
@@ -0,0 +1,5 @@
+const PersDataProt = () => {
```

Comment:
don't be afraid and use full name - now you know what it is, but in few weeks you will be like, wtf is that Prot in name ;) it's also good for other developers to read and understand code

Reply:
Name changed.